cssEditor.html

```diff
@@ -0,0 +1,24 @@
+<html>
+    <head>
+        <title> CSS Editor </title>
+        <link rel="stylesheet" type="text/css" href="css/cssEditor.css">
+        <link rel="stylesheet" href="https://stackpath.bootstrapcdn.com/bootstrap/4.3.1/css/bootstrap.min.css" integrity="sha384-ggOyR0iXCbMQv3Xipma34MD+dH/1fQ784/j6cY/iJTQUOhcWr7x9JvoRxT2MZw1T" crossorigin="anonymous">
+    </head>
+    <body onkeyup="starter()">
```

Comment:
We can add `onkeyup="starter()"` in the **cssEditor.js** file using `addEventListner`. It will make this HTML file more cleaner. Also, It will help us to organize the functionality.